docs/06-concepts/11-authentication/04-providers/06-custom-providers.md

```diff
@@ -140,7 +179,58 @@ await AuthKey.db.deleteWhere(
 );
 ```
 
-In the above example, all auth tokens associated with the user `userId` and created with the method `username` are removed from the `AuthKey` table.
+> ⚠️ **Warning**: Directly removing authentication tokens from the `AuthKey` table bypasses necessary processes such as closing streaming connections and synchronizing server state. It is strongly recommended to use `UserAuthentication.revokeAuthKey` to ensure a complete and consistent sign-out.
```

Comment:
Suggestion: This is not formatted a s a proper warning when you run the documentation.